src/main/java/io/jenkins/plugins/orka/AgentTemplate.java

```diff
@@ -92,20 +93,31 @@ public AgentTemplate(String vmCredentialsId, String vm, boolean createNewVMConfi
         this(vmCredentialsId, vm, createNewVMConfig, configName, baseImage, numCPUs, numExecutors, remoteFS,
                 mode, labelString, namePrefix, retentionStrategy, verificationStrategy, nodeProperties, jvmOptions,
                 scheduler, "auto");
-
     }
 
     @DataBoundConstructor
```

Comment:
you should remove this. DataBound is applied only to the ctor that is created from the UI